src/backend/joanie/tests/core/test_api_admin_users.py

```diff
@@ -211,3 +211,156 @@ def test_admin_api_user_delete(self):
+
+        response = self.client.get("/api/v1.0/admin/users/me/")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.json(),
+            {
+                "id": str(admin.id),
+                "username": admin.username,
+                "full_name": admin.get_full_name(),
+                "is_superuser": True,
+                "is_staff": True,
+                "abilities": {
+                    "delete": False,
+                    "get": True,
+                    "patch": True,
+                    "put": True,
+                    "has_course_access": False,
+                    "has_organization_access": False,
+                },
+            },
+        )
+
+    def test_admin_api_user_me_course_access(self):
+        """
+        User should see their infos with the correct accesses on the me route
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+        factories.UserCourseAccessFactory(user=admin)
+        response = self.client.get("/api/v1.0/admin/users/me/")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.json(),
+            {
+                "id": str(admin.id),
+                "username": admin.username,
+                "full_name": admin.get_full_name(),
+                "is_superuser": True,
+                "is_staff": True,
+                "abilities": {
+                    "delete": False,
+                    "get": True,
+                    "patch": True,
+                    "put": True,
+                    "has_course_access": True,
+                    "has_organization_access": False,
+                },
+            },
+        )
+
+    def test_admin_api_user_me_organization_access(self):
+        """
+        User should see their infos with the correct accesses on the me route
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+        factories.UserOrganizationAccessFactory(user=admin)
+        response = self.client.get("/api/v1.0/admin/users/me/")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.json(),
+            {
+                "id": str(admin.id),
+                "username": admin.username,
+                "full_name": admin.get_full_name(),
+                "is_superuser": True,
+                "is_staff": True,
+                "abilities": {
+                    "delete": False,
+                    "get": True,
+                    "patch": True,
+                    "put": True,
+                    "has_course_access": False,
+                    "has_organization_access": True,
+                },
+            },
+        )
+
+    def test_admin_api_user_me_not_staff(self):
```

Comment:
maybe to avoid this test you can just do on the previous tests:
```python
admin = factories.UserFactory(is_staff=random.choice([True, False]), is_superuser=random.choice([True, False]))
```
So that we monitor the fact that these 2 fields don't impact what we are testing without duplicating to many tests?